packages/terra-dialog/src/Dialog.jsx

```diff
@@ -3,39 +3,60 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import 'terra-base/lib/baseStyles';
 import ContentContainer from 'terra-content-container';
+import ActionHeader from 'terra-clinical-action-header';
+import DialogBody from './DialogBody';
+import DialogFooter from './DialogFooter';
 import styles from './Dialog.scss';
 
 const cx = classNames.bind(styles);
 
 const propTypes = {
- /*
- * The header element to be placed within the header area of the dialog.
- */
-  header: PropTypes.node.isRequired,
-  /*
- * The footer element to be placed within the footer area of the dialog.
- */
-  footer: PropTypes.node,
-  /*
- * The children to be placed within the main content area of the dialog.
- */
+  /**
+   * The text string to be placed within the header area of the dialog.
+   */
+  header: PropTypes.string.isRequired,
+  /**
+   * The footer element to be placed within the footer area of the dialog.
+   * The footer content area will not display if this prop is not provided. UNDER EVALUATION
+   */
+  footer: PropTypes.node.isRequired,
```

Comment:
Remove the isRequired part, since it doesn't sound like it's required.

Reply:
Actually, I forgot to update the doc. The footer is required now.

Reply:
https://github.com/cerner/terra-core/pull/1209/commits/2fabef8d09a87699422ef4b44f00ce87220ae782